src/chrome/content/rules/Criosweb.xml

```diff
@@ -35,12 +36,12 @@
 		<test url="http://luci.criosweb.ro/wp-content/uploads/2015/11/board-config.png" />
 		<test url="http://luci.criosweb.ro/wp-includes/js/tw-sack.min.js" />
 
-	<!--	http://criosweb.ro/$ auto redirects to http://www.criosweb.ro/$ by serve.	-->
-	
+
+	<!--	Directly	-->
 	<target host="criosweb.ro" />
 	<target host="www.criosweb.ro" />
 
-	<rule from="^http://(www\.)?criosweb\.ro/" to="https://www.criosweb.ro/" />
+	<rule from="^http:" to="https:" />
```

Comment:
This would also affect the hosts above.
You have to explicit redirect `(www\.)?criosweb\.ro` to `$1criosweb.ro`.